__init__.py

```diff
@@ -84,6 +85,7 @@ def currency_filter(num, grouping=True):
     return locale.currency(num, grouping=grouping)
 
 if __name__ == '__main__':
+    nr_agent.initialize()
```

Comment:
My impression from the docs is that we can either run our app prefixed with `newrelic-admin run-program` or import and initialize the agent within the app. One or the other should be enough--I don't think we need both.

https://docs.newrelic.com/docs/agents/python-agent/installation-configuration/python-agent-integration